logstash-core/lib/logstash/api/lib/app/commands/system/basicinfo_command.rb

```diff
@@ -8,8 +8,20 @@ def run
     {
       "hostname" => hostname,
       "version" => {
-        "number" => LOGSTASH_VERSION
+        "number" => LOGSTASH_VERSION,
+        "logstash-core" => {
+          "version" => {
+            "number" => fetch_logstash_core_version
```

Comment:
Hey, thinking about this more, lets keep this simple for now. LS Core and LS Event Versions are the same. If and when it diverges, we can display it. Sorry I'd like this to be super easy for users to understand. Having multiple versions just confuses users.

@purbon Lets go back to 

```
{
        "hostname" => hostname,
        "version" => {
           "number" => "2.2.0"
           "build_hash => "72cd1f1a3eee09505e036106146dc1949dc5dc87"
    }
}

```